### DISCUSSION TITLE
이거 풀게요 12월 세번째 주

### DISCUSSION THREAD

**Discussion by brewagebear** [brewagebear/ISOLVEDTHEPROBLEM#12] [금주의 문제]
**금주의 챌린지 문제** 

[크레인 인형뽑기 게임](https://programmers.co.kr/learn/courses/30/lessons/64061)
[실패율](https://programmers.co.kr/learn/courses/30/lessons/42889)

**개인적으로 풀 문제(참가자 알아서 임의의 문제 설정)**

[없는 숫자 더하기](https://programmers.co.kr/learn/courses/30/lessons/86051)
[음양 더하기](https://programmers.co.kr/learn/courses/30/lessons/76501)

**Comment by csj4032** [comment]
https://programmers.co.kr/learn/courses/30/lessons/64061 크레인 인형뽑기 게임
https://programmers.co.kr/learn/courses/30/lessons/42889 실패율

**Comment by pch8388** [comment]
금주의 챌린지 문제

[크레인 인형뽑기 게임](https://programmers.co.kr/learn/courses/30/lessons/64061)
[실패율](https://programmers.co.kr/learn/courses/30/lessons/42889)

개인적으로 풀 문제

**Comment by Tianea2160** [comment]
https://programmers.co.kr/learn/courses/30/lessons/67256 키패드 누르기
https://programmers.co.kr/learn/courses/30/lessons/12901 2016년

**Comment by tate-git** [comment]
금주의 챌린지 문제
[크레인 인형뽑기 게임](https://programmers.co.kr/learn/courses/30/lessons/64061?language=java)
[실패율](https://programmers.co.kr/learn/courses/30/lessons/42889)
